### PR TITLE
Dev/93 When create a new problem with title, then a new item with empty problem name is shown.

### PR DESCRIPTION
**Pull request by nyngwang**
When creating a new problem with a title,  then a new item with an empty problem name is shown

Expected: the new item should show the problem's title 

Addition: 

1. also, should go to the edit page after create  a new problem **successfully**
2. should go to the exam page after create a new exam **successfully**

------------

There is no warnings on the browser console :).

### CODE REVIEW

File: src/app/commons/UX/ScrollToTop.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {useLocation} from "react-router-dom";
+import {useEffect} from "react";
+
+
+export const ScrollToTop = () => {
+    const { pathname } = useLocation()
```
Comment (Johnny850807) [inline src/app/commons/UX/ScrollToTop.js:6]: add a comment to explain why `pathname` is necessary

File: src/app/exam/ExamInPageNavigationBar.js [hunk 2]
```diff
@@ -3,15 +3,16 @@ import {AiOutlineSetting, FaClipboardList, FaRegEdit, FaUserFriends} from "react
...
+    const resetFunctions = [setCurrentExam, setShouldRedirect]
 
     return (
         <InPageNavigationBar currentURL={currentURL}
                              path={{
                                  head: "Exam",
                                  tail: examName
                              }}
-                             events={{ firstParamResetState }}
+                             events={{ firstParamOnClicks: resetFunctions }}
```
Comment (Johnny850807) [inline src/app/exam/ExamInPageNavigationBar.js:15]: 1. why not put all events as properties (props) of `InPageNavigationBar`

2. `firstParamOnClicks` .. has any other better names?  `onCrumbClick`
where `index` is passed to `onCrumbClick` as `onCrumbClick(index)`

**Review by Johnny850807 — CHANGES_REQUESTED** [review]
Review done

File: src/app/exam/ExamInPageNavigationBar.js (hunk 2 above)
Comment (nyngwang) [reply src/app/exam/ExamInPageNavigationBar.js:15]: This is indeed a nice catch, thank u! :)

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Pull request merged**